packages/shorebird_cli/lib/src/commands/release/release_command.dart

```diff
@@ -263,6 +271,18 @@ of the iOS app that is using this module. (aar and ios-framework only)''',
       throw ProcessExit(ExitCode.software.code);
     }
 
+    // If a user explicitly specified --build-name (and optionally
+    // --build-number) We ensure that the version is releasable to avoid
+    // unnecessarily waiting for a build.
```

Comment:
```suggestion
    // If a user explicitly specified --build-name (and optionally
    // --build-number), we ensure that the version is releasable to avoid
    // unnecessarily waiting for a build.
```